spec/pronto/formatter/github_formatter_spec.rb

```diff
@@ -25,7 +27,7 @@ module Formatter
               .should_receive(:create_commit_comment)
               .once
 
-            subject.should eq format(published_comments_msg, count: 1)
+            subject.should eq format(published_comments_msg, count: 1, existing_count: 1)
```

Comment:
I'm not sure this makes sense, as it means that 1 comments added, and 1 existing. I'd rather expect 0 existing. Maybe we need to use `comments.count - additions.count` instead?

Reply:
Originally I tried to use `existing` comments, will re-check once more, maybe I missed something